src/main/java/lotto/domain/rank/Rank.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain.rank;
+
+import static java.util.Arrays.stream;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum Rank {
+    FIRST(1, 6, false, 2000000000),
```

Comment:
```suggestion
    FIRST(1, 6, false, 2_000_000_000),
```
와 같이 가독성을 높일 수 있답니다.